### PR TITLE
Taurus -> GEMD migration

### PR DESCRIPTION
**Pull request by kroenlein**
# Citrine Python PR
This PR is intended to
* Modify all requirements so that installs rely on gemd-python not citrine-taurus
* Update all in-library reference from Taurus to GEMD to avoid spurious deprecation warnings

## Description 
As described above.  This is motivated by impending roll-outs to a customer site.

I am less than confident in the corrections to Pipfile and Pipfile.lock, so extra care should be taken about verifying those edits.

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [ ] I have bumped the version in setup.py

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ requests-mock = "*"
 
 [packages]
 citrine = {editable = true,path = "."}
-taurus = {git = "https://github.com/CitrineInformatics/taurus.git", editable = true, ref="develop"}
+gemd = {git = "https://github.com/CitrineInformatics/gemd-python.git", editable = true, ref="develop"}
```
Comment (bfolie) [inline Pipfile:19]: It looks like gemd doesn't have a develop branch. Did it used to have one? In that case I think it should be changed to `ref="master"`

File: Pipfile.lock [hunk 2]
```diff
@@ -199,10 +199,10 @@
             ],
             "version": "==0.1.7"
         },
-        "taurus": {
+        "gemd": {
             "editable": true,
-            "git": "https://github.com/CitrineInformatics/taurus.git",
-            "ref": "e043283fab2f18112d8e71f102657527fc09efe9"
+            "git": "https://github.com/CitrineInformatics/gemd-python.git",
+            "ref": "1733aa751683b915666f30a24ca289cfcf9da305"
```
Comment (bfolie) [inline Pipfile.lock:205]: Did you run `pipfile lock` or did you copy a git hash?

File: Pipfile (hunk 1 above)
Comment (kroenlein) [reply Pipfile:19]: Yeah, it got moved into masterflow as well.  Good catch.

Comment (kroenlein) [reply Pipfile:19]:
```suggestion
gemd = {git = "https://github.com/CitrineInformatics/gemd-python.git", editable = true, ref="master"}
```

File: Pipfile.lock (hunk 2 above)
Comment (kroenlein) [reply Pipfile.lock:205]: Manually copied the git hash out of the 0.6.2 release, as tagged in PyPi.

**Pull request merged**